Sources/ArcGISToolkit/Components/FeatureFormView/AttachmentHelpers/AttachmentPhotoPicker.swift

```diff
@@ -53,7 +53,8 @@ struct AttachmentPhotoPicker: ViewModifier {
                     
                     newAttachmentImportData = AttachmentImportData(
                         data: data,
-                        contentType: mimeType ?? "application/octet-stream"
+                        contentType: mimeType ?? "application/octet-stream",
+                        fileExtension: item.supportedContentTypes.first?.preferredFilenameExtension ?? ""
```

Comment:
The doc of `supportedContentTypes` says:
> The content types the item supports in order of the most preferred to the least.

So, I think using the first item should be safe.